### DISCUSSION TITLE
KnowledgeBase: Simple website to track technologies

### DISCUSSION THREAD

**Discussion by ratnadeep007** [pocketbase/pocketbase#1844] [Show and tell]
Hey everyone,

I created a simple website using Astro.js and Pocketbase to keep track of technologies, tools and blog/articles. Its like my personal diary to keep things related to tech. 

Deployed Pocketbase in fly.io following [Host for free on Fly.io #537](https://github.com/pocketbase/pocketbase/discussions/537).
Used Astro.js to build frontend and Tailwind for CSS and deployed it to Vercel using there Vercel Edge.

Have a look:

[Knowledge Base](https://kb.theskylab.in/)

[Project Link - Github](https://github.com/ratnadeep007/KnowledegeBase)

Contributions are welcome. 😊

**Comment by danfascia** [comment]
Very nice - it would be really cool to add in pocketbase authentication, not because you need it for this site necessarily but for the example

**Comment by techiemon** [comment]
Smooth, thanks for the example!   I'm taking a crash course in Astro and was wondering if PB worked fine for the static content.  Nice work.
